web/models.py

```
@@ -2804,3 +2805,20 @@ class ScheduledPost(models.Model):
 
     def __str__(self):
         return self.content
+
+
+def default_valid_until():
```

Ruff (0.8.2)

Missing return type annotation for public function `default_valid_until`

(ANN201)